migrations/0045-fix-discord-gcal-field.ts

```diff
@@ -0,0 +1,12 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class FixDiscordGcalField1714770061929 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query('ALTER TABLE "Events" ALTER COLUMN "discordEvent" TYPE varchar');
+    await queryRunner.query('ALTER TABLE "Events" ALTER COLUMN "googleCalendarEvent" TYPE varchar');
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    // nothing here because it's fixing an earlier migration (# 0044)
```

Comment:
Despite this being a fix, I believe it is still a good practice to alter the table type back to what it was originally.